src/components/Footer/styled.js

```diff
@@ -2,7 +2,6 @@ import Typography from '@mui/material/Typography';
 import { styled } from "@mui/material/styles";
 
 export const StyledFooter = styled(Typography)`
```

Comment:
It should be as Box element, not Typography, but inside component you should place Typography into StyledFooter